contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,651 @@
+        </tr>
+        <tr>
+            <td><strong>Funnels</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Understand conversion between events, pages</td>
+        </tr>
+        <tr>
+            <td><strong>Cohorts</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Combine users based on properties and events for group analysis</td>
+        </tr>
+        <tr>
+            <td><strong>User paths</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track user flows and where they drop-off</td>
+        </tr>
+        <tr>
+            <td><strong>Retention</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Analyze user and revenue retention</td>
+        </tr>
+        <tr>
+            <td><strong>UTM tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track marketing campaigns with UTM tags</td>
+        </tr>
+        <tr>
+            <td><strong>Correlation analysis</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Suggested events and properties that lead to success or failure</td>
+        </tr>
+        <tr>
+            <td><strong>Group analytics</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Understand how organizations use your product</td>
+        </tr>
+        <tr>
+            <td><strong>Formulas</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Use custom formulas to calculate unique insights</td>
+        </tr>
+        <tr>
+            <td><strong>SQL access</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Write your own queries in SQL</td>
+        </tr>
```

Comment:
Is there anything Pendo has that we don't? Maybe retro active analytics?

Reply:
Nvm retroactive analytics is their name for autocapture 🤔 https://www.pendo.io/pendo-blog/future-retro-one-early-product-decision-made-difference-pendo/